functions/utils/getChickens.js

```diff
@@ -21,6 +21,7 @@ const potentialChickens = [
   { id: "U0274DF37", name: "Mac" },
   { id: "U0K8Z1KD1", name: "Melissa" },
   { id: "U44G2GD4M", name: "Usama" },
+  { id: "U44G2GD4M", name: "Damien" },  
```

Comment:
same id ?

Reply:
`id` is currently irrelevant - I think it's for a future feature with slack ids.